### ISSUE TITLE
rio run --config option adding word "config" to my target on k8s side?

### ISSUE THREAD

**Issue by tfiduccia** [rancher/rio#366]
Steps:
1.  rio run -n nsp/test1 --config foo:bar nginx
2. kubectl get -n nsp -o=json deployment/test1

Results:  If you look in the volumeMounts section you will see the word "config" got added to my target name.  Also the mountPath never got added to the volume section?

![image](https://user-images.githubusercontent.com/11514927/62656104-581a8f00-b918-11e9-9a81-09a9fd4ab5b8.png)

![image](https://user-images.githubusercontent.com/11514927/62656221-8b5d1e00-b918-11e9-99a7-4f5eb4a981d1.png)

**Comment by StrongMonkey** [comment]
MountPath is not supposed to show up in volume section. Any bug you are hitting?

**Comment by davidnuzik** [comment]
@rancher-max please review as low priority; may have just been config/yaml issue? Probably can just close this?

**Comment by rancher-max** [comment]
Results are exactly the same in v0.6.0. Bug is that the deployment never becomes ready, so the pod is stuck in "PodInitializing" and the rio service never hits readystate.

**Comment by davidnuzik** [comment]
@StrongMonkey sounds like needs investigation in v0.6.0 or touch base with Max.

**Comment by StrongMonkey** [comment]
@rancher-max You have to create a configmap foo in the same namespace, otherwise the pod won't start.

**Comment by rancher-max** [comment]
Okay so that seemed to work okay. Steps I took:
1. `rio -n nsp config create -k foo test1 /path/to/test.properties`
2. `rio inspect configmap/nsp/test1` ==> 
```
---
apiVersion: v1
data:
  foo: |-
    foo=bar
    test=whatever
kind: ConfigMap
metadata:
  creationTimestamp: "2019-11-01T18:35:34Z"
  name: test1
  namespace: nsp
  resourceVersion: "1699939"
  selfLink: /api/v1/namespaces/nsp/configmaps/test1
  uid: 5679af70-fb90-4215-8741-0dc9ff4f209f
```
3. rio run -n nsp/testing1 -p 80/http --config test1:foo nginx

`kubectl get -n nsp -o=json deployment/testing1` has the same output as in the original issue, but there don't appear to be any errors.

Futher, running: `rio -n nsp exec -it -c testing1 testing1 cat foo/foo` produces the expected value:
```
foo=bar
test=whatever
```

I'm calling this fixed in v0.6.0-alpha.2
